pkg/services/network/constants.go

```diff
@@ -22,6 +22,10 @@ const (
 	NSXTTypeNetwork = "nsx-t"
 	// NSXTVNetSelectorKey is also defined in VM Operator.
 	NSXTVNetSelectorKey = "ncp.vmware.com/virtual-network-name"
+	// NSXVPCTypeNetwork is the name of the NSX VPC network type.
+	NSXVPCTypeNetwork = "nsx-t-subnetset"
```

Comment:
We should either rename the value or the key to match.

So either `NSXTSubnetSetTypeNetwork` or `NSXTVPCTypeNetwork = "nsx-t-vpc"`